fastlane/Helpers/Match.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Match.swift
+//  FastlaneRunner
+//
+//  Created by Su Ho on 22/09/2022.
+//  Copyright © 2022 Joshua Liebowitz. All rights reserved.
+//
+
+final class Match {
+
+    static func syncCodeSigning(type: MatchType, appIdentifier: [String], isForce: Bool = false) {
+        if isCi() {
+            Keychain.create()
+            match(
+                type: type.value,
+                readonly: .userDefined(!isForce),
+                appIdentifier: appIdentifier,
+                username: .userDefined(Constant.userName),
+                teamId: .userDefined(Constant.teamId),
+                gitUrl: Constant.matchURL,
+                keychainName: Constant.keychainName,
+                keychainPassword: .userDefined(Secret.keychainPassword),
+                force: .userDefined(isForce)
+            )
+        } else {
+            match(
+                type: type.value,
+                readonly: .userDefined(!isForce),
+                appIdentifier: appIdentifier,
+                username: .userDefined(Constant.userName),
+                teamId: .userDefined(Constant.teamId),
+                gitUrl: Constant.matchURL,
+                force: .userDefined(isForce)
+            )
+        }
+    }
+}
+
+extension Match {
+
+    enum MatchType: String {
+
+        case development
+        case adHoc = "adhoc"
```

Comment:
I think we dont have to use camel case for `adHoc` and `appStore`. Or is there any reason behind this?

Reply:
@ducbm051291 We should use English when defining enum cases.
So we have:
- Ad hoc
- App Store

Reply:
`match` is the only one that use `ad hoc` as `adhoc`  😆